src/src/Helpers/HTTPResponseParser.h

```diff
@@ -3,16 +3,28 @@
 
 #include "../../ESPEasy_common.h"
 #include "../../_Plugin_Helper.h"
+#include "../Helpers/Networking.h"
```

Comment:
Here (after the include of _Plugin_Helper.h) you should also have a check for `#if RESPONSE_PARSER_SUPPORT`
So there isn't a function declaration being made if there also isn't a function body in the .cpp